.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 src/metabase @camsaul
 modules @camsaul
-enterprise/backend @camsaul
+enterprise/backend @noahmoss
```

Comment:
There's more stuff in `enterprise/backend` than just permissions. The Sandboxing QP stuff is one example

Reply:
advanced_config → noah
advanced_permissions → noah
api → cam
audit_app → noah
billing → noah
content_verification → cam
dashboard_subscription_filters → cam
llm → cam (not that it's real)
sandbox → noah except sandbox/query_processor → cam
search → cam
serialization → cam unless you want to extend this to braden and sanya already
snippet_collections → noah
sso → noah
task → noah (it only has truncate_audit_tables.clj)
*.clj files → cam, except internal_user.clj, that's noah

please list the stuff where you disagree and why, thanks

Reply:
I don't think `advanced_config` has anything to do with perms or audit, and I wrote that stuff, so it probably doesn't make sense to move that as part of this

Reply:
I thought we were just talking about permissions when we discussed this the other day, I guess audit stuff and SSO could make sense too, altho I don't remember discussing that. I don't think we should move other stuff like advanced config over at this point in time